swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+
 class Item:
-    pass
+    #category is an empty string 
+    def __init__(self, category = None, condition = 0):
+        if not category:
+            category = ""
+        self.category = category
+        self.condition = condition  
+        
+    def __str__(self):
+        return ("Hello World!")
+    
+    ### WAVE 5 ###
+
+    def condition_description(self):
+        if self.condition == 0:
```

Comment:
Consider how you can use >= rather than == to account for fractional values of the condition.